aria/model/model.py

```diff
@@ -102,7 +114,15 @@ def __init__(self, model_config: ModelConfig):
         self.max_seq_len = model_config.max_seq_len
 
         # Positional embeddings
-        self.rotary_emb = RotaryEmbedding(self.d_head)
+        if use_yarn:
+            # todo: need more testing on this
+            self.rotary_emb = DynamicYaRNScaledRotaryEmbedding(self.d_head,
```

Comment:
This import is missing.

Reply:
Oh yes you are right. Will fix it in a moment.